### PR TITLE
Add basic refund feature

### PR DESCRIPTION
**Pull request by DenisCarriere**
## Add basic refund feature

Some users have experienced their funds being stuck after the +3 day unstaking period.

A basic refund feature has been added.

## EOSIO Contract

- [`eosio.system::refund`](https://github.com/EOSIO/eos/blob/master/contracts/eosio.system/eosio.system.abi#L179-L185)

## Future To-Dos (not in this PR)
- [ ] Status of Refund (time counter)

## Images of Refund Feature

image

image

### CODE REVIEW

File: app/components/RefundForm/tests/index.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('<RefundForm />', () => {
```
[the remark this answers is not in the document]
Comment (QuentinDejean) [reply app/components/RefundForm/tests/index.test.js]: As per the discussion on Telegram channel, let's discard above comment and hold on until the testing framework is it a good state (currently really outdated)

[the remark this answers is not in the document]
Comment (andymeek) [reply app/components/RefundForm/tests/index.test.js]: I'd remove the file as it's a pointless failing test.

[the remark this answers is not in the document]
Comment (DenisCarriere) [reply app/components/RefundForm/tests/index.test.js]: 👍 Cool, removed tests entirely

File: app/containers/Refund/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { makeSelectEosAccount } from 'containers/Scatter/selectors';
+
+import injectSaga from 'utils/injectSaga';
+import injectReducer from 'utils/injectReducer';
+import reducer from './reducer';
+import saga from './saga';
+import submitAction from './actions';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export class Refund extends React.Component {
```
[the remark this answers is not in the document]
Comment (QuentinDejean) [reply app/containers/Refund/index.js]: Awesome, thanks mate!

Yes no worries, my first PR was to check whether we were all on the same page. Will action that

**Pull request merged**